src/main/java/FigureSupplier.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        String[] nameFigure = new String[]{"circle", "square", "rectangle",
+                "right triangle", "isosceles trapezoid"};
+        int randomShape = new Random().nextInt(nameFigure.length);
+        switch (nameFigure[randomShape]) {
+            case "circle":
+                return new Circle(nameFigure[randomShape]);
+            case "square":
+                return new Square(nameFigure[randomShape]);
+            case "rectangle":
+                return new Rectangle(nameFigure[randomShape]);
+            case "right triangle":
+                return new Triangle(nameFigure[randomShape]);
+            case "isosceles trapezoid":
+                return new Trapezoid(nameFigure[randomShape]);
+            default:
+                return null;
```

Comment:
checklist: Don’t return null from a method.
Returning null from a method is a bad practice. If you use switch-case in your solution, you may just put the last possible option in the default case.